rmw_fastrtps_shared_cpp/include/rmw_fastrtps_shared_cpp/topic_cache.hpp

```diff
@@ -214,18 +215,29 @@ inline std::ostream & operator<<(
 }
 
 template<class T>
-class LockedObject : public T
+class LockedObject
 {
 private:
-  mutable std::mutex cache_mutex_;
+  mutable std::mutex mutex_;
```

Comment:
What prompted the name change from `cache_mutex_` to `mutex_`?

Reply:
Because it's a generic class with a `T object_` that could theoretically be anything, not just a cache. It seemed more accurate

Reply:
As an outsider to this package that makes sense to me.